game-runner/src/main/java/za/co/entelect/challenge/player/bootstrapper/PlayerBootstrapper.java

```diff
@@ -68,7 +68,7 @@ private Player parsePlayer(String playerConfig, String playerNumber, GameRunnerC
         if (playerConfig.equals("console")) {
             player = new ConsolePlayer(String.format("BotPlayer %s", playerNumber));
         } else {
-            LOGGER.info(playerConfig);
+            LOGGER.info(String.format("Config for player %s: %s", playerNumber, playerConfig));
```

Comment:
I don't think you need String.format here:

```
LOGGER.info("Config for player {} : {}", playerNumber, playerConfig)
```

One of the reasons for using the logger instead of String.format is that String.format will always be executed whereas the logger's format will only be executed if it is logging at info level.

Reply:
done :)